packages/profiling-node/src/integration.ts

```diff
@@ -293,6 +308,26 @@ class ContinuousProfiler {
     chunk.timer.unref();
   }
 
+  /**
+   * Attaches profiling information to spans that were started
+   * during a profiling session.
+   */
+  private _setupSpanChunkInstrumentation(): void {
+    getGlobalScope().setContext('profile', {
+      profiler_id: this._profilerId,
+      ['thread.id']: PROFILER_THREAD_ID_STRING,
+      ['thread.name']: PROFILER_THREAD_NAME,
```

Comment:
can we double check this works as expected with worker threads (albeit not sure how profiling works with worker threads today).

Reply:
I'll need to change the PR here btw.  We'll need to assign thread_id and thread_name on the context.trace.data. The plan was to do it from https://github.com/getsentry/sentry-javascript/blob/develop/packages/core/src/utils/applyScopeDataToEvent.ts#L165, would that be the right place to do it?

Profiling doesnt support worker threads, you'd basically need to reinitialize the SDK and the profiler there to get it working, but there wouldnt be any system linking. It would just treat it as an isolated instance

Reply:
@AbhiPrasad I changed the implementation to only assign this to profiled events from within the integration